discovery-infra/test_infra/helper_classes/config/controller_config.py

```diff
@@ -0,0 +1,33 @@
+from abc import ABC
+from pathlib import Path
+from dataclasses import dataclass
+
+from test_infra import consts
+from .base_config import _BaseConfig
+
+
+@dataclass
+class ControllerConfig(_BaseConfig, ABC):
+    platform: str = None
+    base_dns_domain: str = None   # Todo - Might change during MGMT-5370
+    is_ipv6: bool = None   # Todo - Might change during MGMT-5370
+    bootstrap_in_place: bool = None
+    private_ssh_key_path: Path = None
+    working_dir: str = consts.WORKING_DIR
```

Comment:
Set to `None` and get set the value on `global_variables` object

Reply:
why? the work dir is constant not an env variable